third_party/terraform/tests/resource_compute_disk_test.go.erb

```diff
@@ -457,9 +460,64 @@ func TestAccComputeDisk_resourcePolicies(t *testing.T) {
 				ImportState:       true,
 				ImportStateVerify: true,
 			},
+			{
+				Config: testAccComputeDisk_resourcePolicies_updated(diskName, policyName2),
```

Comment:
I think you want to reuse `policyName` here, so that we're updating the resource instead of making a new one?